javalib/src/main/scala/java/util/LinkedList.scala

```diff
@@ -41,7 +41,7 @@ class LinkedList[E]()
       peekLast()
   }
 
-  def removeFirst(): E = {
+  override def removeFirst(): E = {
```

Comment:
This seems like it was added by mistake, among all the `remove`s.

Reply:
Oops!   Thank you for finding it, fixed now.